includes/class-sensei-pro-upsell.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * File containing the Sensei_Pro_Upsell class.
+ *
+ * @package sensei-lms
+ * @since   4.23.0
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Class Sensei_Pro_Upsell
+ *
+ * Handles the generation of Pro upsell URLs.
+ *
+ * @since 4.24.6
```

Comment:
Same here, the version should be `$$next-version$$`.

Reply:
I got the same feedback from Renatho, and I didn't know about the special tag, so I manually entered the next version. :)

It's replaced now. Thanks

Reply:
Haha, yeah, it's a bit confusing practice. :)

Thanks for the updates!